### ISSUE TITLE
error while trying to load model2.

### ISSUE THREAD

**Issue by oxfordhalfblood** [d-acharya/CovPoolFER#11]
I'm trying to finetune the model2 from the Googledrive uploaded models and while trying to use error is coming as: 
"Key CovPool2/Bottleneck/BatchNorm/beta not found in checkpoint"

Also, in the covpoolnet2.py there is dlp_cnn() . But there is no dlp_cnn() function in the whole repository.

But on your Notes you wrote you are using model1 although there isn't any model1 inside the models folder.
